apisix/discovery/kubernetes/init.lua

```diff
@@ -431,7 +431,7 @@ local function single_mode_init(conf)
 
     local default_weight = conf.default_weight
     local endpoints_informer, err
-    if conf.watch_endpoint_slices_schema then
+    if conf.watch_endpoint_slices then
```

Comment:
```suggestion
    if conf.watch_endpoint_slices then
```

I don't understand why would you need that? What is the problem it is trying to solve here?